src/views/account/News/Atoms/Item.tsx

```diff
@@ -94,4 +110,13 @@ const NewsListItem: React.FC<NewsListItemProps> = ({ index, message, navigation,
   );
 };
 
+const styles = StyleSheet.create({
+  newsImage: {
+    width: "100%",
+    height: 200,
+    borderRadius: 8,
```

Comment:
Faudrait enlever le radius en bas de l'image ça fait un peu bof je trouve.
```suggestion
    borderTopRightRadius: 8,
    borderTopLeftRadius: 8,
```